File: problems/27/solution.cs
public class Solution {
    public int RemoveElement(int[] nums, int val) {
        var k = 0;
        
        for (var i = 0; i < nums.Length; i++) {
            if (nums[i] != val) {
                nums[k] = nums[i];
                k++;
            }
        }
        
        return k;
    }
}

File: problems/27/solution.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        i: int = 0
            
        while i < len(nums):
            if nums[i] == val:
                nums.remove(nums[i])
            else:
                i += 1
        
        return len(nums)